app/screens/post_options/index.test.js

```diff
@@ -69,12 +71,24 @@ describe('makeMapStateToProps', () => {
         expect(props.canFlag).toBe(true);
     });
 
-    test('canMarkAsUnread is true when isMinimumServerVersion is 5.18v', () => {
+    test('canMarkAsUnread is true when isMinimumServerVersion is 5.18v and channel not archived', () => {
+        channelSelectors.getChannel = jest.fn().mockReturnValueOnce({
```

Comment:
test still should check for the minimum server version

Reply:
Got it. I added minimum server version checks as well as a fourth test case to be explicit.